src/kudu/consensus/consensus_queue.cc

```diff
@@ -999,8 +999,62 @@ void PeerMessageQueue::AdvanceMajorityReplicatedWatermarkFlexiRaft(
 }
 
 std::map<std::string, int> PeerMessageQueue::GetDataCommitQuorum() {
+  // Map to store the number of voters in each region from the active config.
+  std::map<std::string, int> voter_distribution;
+
+  // Return value denoting the number of votes required in each region.
   std::map<std::string, int> data_commit_quorum;
```

Comment:
the data_commit_quorum is an output variable and by kuduraft convention should be passed in as ptr.
The function can return either Status or bool for success/failure.